src/prepare-app-options/param-separator.ts

```diff
@@ -0,0 +1,2 @@
+const REGEX_PATH_SEPARATOR = /\s*,\s*/;
```

Comment:
Yes. This might be dead code. Let me check... I'll 🔥 it if it is dead code. Previously I had the separator as `" "` but since I was able to improve the regex for colours and Modelviewer Attributes replace `,` with a `+` when the search params are initialized. Colours uses `,` for RGBA colours but the regex for colour parsing handles that.